Testing/Exec/ReactEval_C/GNUmakefile

```diff
@@ -41,12 +46,16 @@ ifeq ($(USE_SUNDIALS_PP), TRUE)
     # provide location of KLU lib if needed
     #SUITESPARSE_DIR=$(PELE_PHYSICS_HOME)/ThirdParty/SuiteSparse/
   endif
-  # Activates GPU version
-  USE_CUDA_SUNDIALS_PP = FALSE
-else
+  ifeq ($(USE_CUDA), TRUE)
+  # Activates GPU versio
+  USE_CUDA_SUNDIALS_PP = TRUE
+  endif
+endif
+
+ifeq ($(USE_RK64_PP), TRUE)
   # Activates use of Hari explicit RK
   # will only work if USE_SUNDIALS_PP = FALSE
-  USE_RK64_PP = FALSE
+  TRANSPORT_CPP = TRUE
```

Comment:
The name of this option makes no sense.

Reply:
yes it does, it's a flag in PP ( also used in PLM for a while) that enables to choose between transport F90 or CPP ...

Reply:
Ok, the +/- here made it look bizarre.